### ISSUE TITLE
Known but unset mathematical properties

### ISSUE THREAD

**Issue by mohamed-barakat** [homalg-project/CAP_project#385]
Is there a reason why `IsIsomorphism` is not automatically set by the corresponding Add-* for CAP operations like:

- IsomorphismFrom*
- LeftUnitor*
- RightUnitor*
- LeftDistributivityExpanding*
- LeftDistributivityFactoring*
- RightDistributivityExpanding*
- RightDistributivityFactoring*
- Braiding*
- BraidingInverse*

**Comment by sebastianpos** [comment]
Logic is usually delegated to the logic files, because otherwise, it would be impossible to switch it off.

So if you like to have these implications, you may add them to the logic files.

**Comment by sebastianpos** [comment]
Ok.

**Comment by heiderich** [comment]
Also there are variants of monoidal categories that do not require LeftUnitor and RightUnitor to be isomorphisms. Same goes for braidings.

**Comment by mohamed-barakat** [comment]
> Also there are variants of monoidal categories that do not require LeftUnitor and RightUnitor to be isomorphisms. Same goes for braidings.

Yes, thanks, I was planning to add corresponding categorical properties.
